justice/lightcurve.py

```diff
@@ -250,9 +250,30 @@ def get_band(cls, conn, dataset, obj_id, band_id):
         times, fluxes, flux_errs, detected = [np.array(series) for series in zip(*cursor.fetchall())]
         return BandData(times, fluxes, flux_errs, detected)
 
+    @classmethod
+    def _get_band_from_blobs(cls, conn, dataset, obj_id, band_id):
+        res = conn.execute('''
+            select mjd, flux, flux_err, detected
+            from {}_blob
+            where object_id = ?
+            and passband = ?
+            '''.format(dataset), [obj_id, band_id]
+        )
+        raw_times, raw_fluxes, raw_flux_errs, raw_detected = res.fetchone()
```

Comment:
jw do you have to close res.close()? nbd

Reply:
if the connection is passed in by the caller, the caller should decide when to close it. if the caller passes in a string and the function creates the connection, the `with` block handles closing it

Reply:
unless i missed something